src/crypto/CHIPCryptoPALPSA.cpp

```diff
@@ -106,113 +106,85 @@ bool isValidTag(const uint8_t * tag, size_t tag_length)
 
 } // namespace
 
+Aes128KeyHandle::~Aes128KeyHandle() = default;
```

Comment:
This does not need to be made explicit

Reply:
I does. I want the compiler to emit the definition here.